templates/pages/assets/socket.html.twig

```diff
@@ -41,20 +41,30 @@
 
    {{ fields.nullField() }}
 
-   {{ fields.numberField(
-      'position',
-      item.fields['position'],
-      __('Position'),
-      field_options
-   ) }}
+   {# Do not display position if several mode, value is compute on add #}
+   {% if params['several'] is not defined %}
```

Comment:
This check is done many times. It could be better to define a variable in template beginning.
```
{% set add_several = params['several'] is defined and params['several'] %}
```
```suggestion
   {% if not add_several %}
```

Reply:
done !